### PR TITLE
Added Fix for retro screen test

### PR DESCRIPTION
**Pull request by KlaudiaLapcionek**

### CODE REVIEW

**Review by MarekPawlowskiQA — APPROVED** [review]
lgtm :)

File: src/test/java/com/intive/template/pages/RetroPage.java [hunk 1]
```diff
@@ -19,12 +16,7 @@
     public boolean openRetroBoardsAndCheckElements() {
```
Comment (MarekPawlowskiQA) [inline src/test/java/com/intive/template/pages/RetroPage.java:16]: Ok, I looked again..... see what happen after removing this code

**Review by MarekPawlowskiQA — CHANGES_REQUESTED** [review]
The for each loop

File: src/test/java/com/intive/template/pages/RetroPage.java [hunk 2]
```diff
@@ -19,12 +16,7 @@
...
-
-            if (!retroScreenTitle.isDisplayed()
-                    || !retroScreenTitle.getText().equals("Board id: " + (index + 1))
-                    || !arrowBackButton.isDisplayed())
-                return false;
-
+            arrowBackButton.isDisplayed();
             arrowBackButton.click();
         }
         return true;
```
Comment (kacperpelc) [inline src/test/java/com/intive/template/pages/RetroPage.java:22]: this method always return true. 'if' statement needed

**Review by kacperpelc — CHANGES_REQUESTED** [review]

File: src/test/java/com/intive/template/pages/RetroPage.java (hunk 2 above)
Comment (KlaudiaLapcionek) [reply src/test/java/com/intive/template/pages/RetroPage.java]: I added that. :)

File: src/test/java/com/intive/template/pages/RetroPage.java (hunk 1 above)
Comment (KlaudiaLapcionek) [reply src/test/java/com/intive/template/pages/RetroPage.java:16]: I added "if" statement here.

**Review by MarekPawlowskiQA — APPROVED** [review]

**Review by kacperpelc — APPROVED** [review]

**Pull request merged**